release.yml

```diff
@@ -127,7 +127,10 @@ stages:
         artifactName: FhirConverterBuild
         downloadPath: $(System.DefaultWorkingDirectory)
     - script: |
-        docker run --rm -d -p 5000:5000 --name registry stefanscherer/registry-windows:2.6.2
+        git clone -q https://github.com/sowu880/dockerfiles-windows.git
```

Comment:
Are we going to maintain our own version of windows image?

Reply:
The public image can not be used for OCI artifacts.